src/commons/config/external-config/external-config.service.ts

```diff
@@ -27,7 +27,7 @@ export class ExternalConfigService
       password: this.appConfigService.get<string>(ENV_KEY.DATABASE_PASSWORD),
       database: this.appConfigService.get<string>(ENV_KEY.DATABASE_DATABASE),
       entities: ['dist/**/*.entity{.ts,.js}'],
-      synchronize: false,
+      synchronize: true,
```

Comment:
```suggestion
      synchronize: false,
```
테스트를 위해 true 로 바꾸신거같은데 github 에는 false 로 유지돼야합니다.

Reply:
넵 pr 날릴때 false로 변경 후 날리겠습니다!